config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] According to the project requirements, kindly make sure to use Postgresql as the database adapter 👍

Reply:
@codepantha thanks for the review, I will make changes to my project now to comply with the project requirement